### PR TITLE
Preset: add "requireSpaceBetweenArguments" rule to all presets

### LINKED ISSUE #940: Add requireSpaceBetweenArguments rule to relevant presets
Would like to add it to a couple of them, including to "wikimedia" 

/cc @Krinkle 

<bountysource-plugin>

---
Want to back this issue? **[Place a bounty on it!](https://www.bountysource.com/issues/7898134-add-requirespacebetweenarguments-rule-to-relevant-presets?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by markelog**
Also corrects link to yandex code-style

Fixes #940

### CODE REVIEW

**Comment by mikesherov** [comment]
LGTM

**Comment by mrjoelkemp** [comment]
+1, though I hope the test data for each preset already has an instance where this rule is used.

**Comment by markelog** [comment]
@mrjoelkemp good point, added

File: test/data/options/preset/wikimedia.js [hunk 1]
```diff
@@ -84,6 +84,7 @@
 		}
 
 		for ( key in item ) {
+            // requireSpaceBetweenArguments
```
Comment (mikesherov) [inline test/data/options/preset/wikimedia.js]: Indent is off here.

Comment (markelog) [inline test/data/options/preset/wikimedia.js]: Already fixed

**Comment by mrjoelkemp** [comment]
I'll land this.

**Pull request closed without merging**